docs/releases/1.11.txt

```diff
@@ -193,6 +193,9 @@ Management Commands
 * The new :option:`loaddata --exclude` option allows excluding models and apps
   while loading data from fixtures.
 
+* The :djadmin:`makemessages` command now requires configured settings, like
```

Comment:
Could it be a backwards-incompatible change under some circumstances? Otherwise, it seems like a bug fix.